src/geometry/shape.rs

```diff
@@ -1,36 +1,32 @@
 use lyon::path::Path;
 use lyon::tessellation::*;
 
-// TODO is this always correct?
-fn bounding_radius(path: Path) -> f32 {
+type PositionBuilder = VertexBuffers<crate::Position, u16>;
+
+fn fill_position(vertex: FillVertex)->crate::Position {
```

Comment:
I guess you could also go `impl From<FillVertex> for crate::Position` if you want fancy. I'm fine either way

Reply:
I could do something like this, but I don't know how I would use this with the rest of the API

```
impl<'a, 'b> From<StrokeVertex<'a, 'b>> for crate::Position {
  fn from(vertex: StrokeVertex<'a, 'b>) -> Self {
    let p = vertex.position();
    crate::Position([p.x, p.y, 0.0])
  }
}

impl<'l> From<FillVertex<'l>> for crate::Position {
  fn from(vertex: FillVertex<'l>) -> Self {
    let p = vertex.position();
    crate::Position([p.x, p.y, 0.0])
  }
}
```